crates/macro-support/src/parser.rs

```diff
@@ -1454,21 +1470,9 @@ impl<'a> MacroParse<(&'a mut TokenStream, BindgenAttrs)> for syn::ItemEnum {
             })
             .collect::<Result<Vec<_>, Diagnostic>>()?;
 
-        let mut values = variants.iter().map(|v| v.value).collect::<Vec<_>>();
-        values.sort();
-        let hole = values
-            .windows(2)
-            .find_map(|window| {
-                if window[0] + 1 != window[1] {
-                    Some(window[0] + 1)
-                } else {
-                    None
-                }
-            })
-            .unwrap_or(*values.last().unwrap() + 1);
-        for value in values {
-            assert!(hole != value);
-        }
+        let hole = (0..u32::MAX)
+            .find(|v| !discriminate_map.contains_key(v))
+            .unwrap_or(u32::MAX);
```

Comment:
If an enum has `u32::MAX` variants, this will output `u32::MAX`, which would be wrong.
Considering that this should be practically impossible, I would argue using `unwrap()` or `expect()` here.

Reply:
Certainly an unlikely edge case, but you're right.

Related question: Should we hard-limit the number of enum variants?

Reply:
> Related question: Should we hard-limit the number of enum variants?

I don't think its necessary, it should actually be quite impossible to make such a big enum to begin with.
In any case, panicking should keep us on the sane side of this thought-experiment.